site/_includes/footer.html

```diff
@@ -48,6 +48,7 @@ <h5>{{ site.footer.title }}</h5>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
     crossorigin="anonymous"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/anchor-js/anchor.min.js"></script>
```

Comment:
Let's use `cdnjs` here so that we don't have to hit a new domain for this. You can also add the `integrity` and `crossorigin` tags

Reply:
Yeah makes sense. Updated.